src/app/routes/storage.js

```diff
@@ -25,6 +25,20 @@ module.exports = (Router, Service, App) => {
   const sharedAdapter = sharedMiddlewareBuilder.build(Service);
   const teamsAdapter = teamsMiddlewareBuilder.build(Service);
 
+  Router.get('/storage/folder/size/:id', passportAuth, sharedAdapter, (req, res) => {
+    const { params, behalfUser } = req;
+    const folderId = params.id;
+    Service.Folder.getSize(behalfUser, folderId)
+      .then((size) => {
```

Comment:
`async-await` could be used here without catching any error as we have a catch-all middleware, thus making code more readable